core/new-gui/src/app/workspace/service/workflow-graph/model/workflow-action.service.ts

```diff
@@ -62,6 +108,8 @@ type GroupInfo = {
   providedIn: "root",
 })
 export class WorkflowActionService {
+  public functionMap: { [key: string]: Function } = {};
```

Comment:
this `functionMap` name is confusing as well. please make it more explicit and meaningful.